app/tests/models/test_vcf_results_model.py

```diff
@@ -0,0 +1,164 @@
+import unittest
+from pydantic import ValidationError
+
+from vep.models.vcf_results_model import (
+    PaginationMetadata,
+    PredictedIntergenicConsequence,
+    PredictedTranscriptConsequence,
+    FeatureType,
+    Strand,
+    ReferenceVariantAllele,
+    Location,
+    AlternativeVariantAllele,
+    Variant,
+    Metadata,
+    VepResultsResponse,
+)
+
+
+class TestVCFResultModel(unittest.TestCase):
+
+    def test_predicted_intergenic_consequence(self):
+        consequence = PredictedIntergenicConsequence(
+            feature_type=None,
+            consequences=["intergenic_variant"]
+        )
```

Comment:
Seeing now for the first time how the `PredictedIntergenicConsequence` class is instantiated, I think passing both these arguments to it is redundant. If the contract is that the `PredictedIntergenicConsequence` class will always serialise to:

```json
{
  "feature_type": null;
  "consequences": ["intergenic_variant"];
};
```

and if the caller of the class always knows that it is calling the `PredictedIntergenicConsequence` class, then there shouldn't be any need to pass any data into it. This class can have the `feature_type` field set always to `None`,  and the `consequences` field set always to `["intergenic_variant"]`.